Changelog.md

```diff
@@ -13,6 +13,7 @@ Compiler Features:
  * Commandline Interface: option ``--pretty-json`` works also with the following options: ``--abi``, ``--asm-json``, ``--ast-compact-json``, ``--devdoc``, ``--storage-layout``, ``--userdoc``.
  * SMTChecker: Support ``abi.encodeCall`` taking into account the called selector.
  * Language Server: Allow full filesystem access to language server.
+ * Language Server: Always adds ``{project_root}/node_modules`` to include search paths.
```

Comment:
I suggest to use `Always add` instead of `Always adds` as it is more consistent with the other entries